src/components/Input/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FormEvent, ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import './styles.css';
+
+interface InputSharedProps {
+    children?: ReactNode;
+    placeholder?: string;
+    onChange?: (event: FormEvent<HTMLInputElement>) => void;
```

Comment:
I think if we're actually doing the input thing now it'd be nice to have it use the custom  `ChangeHandler` type that we use for all the non-standard inputs.

https://github.com/aiera-inc/aiera-client-sdk/blob/d08e6790bea8989b8634dd7ad0bdb32f39a6c4a4/src/types/index.ts#L28

Example uses: https://github.com/aiera-inc/aiera-client-sdk/blob/d08e6790bea8989b8634dd7ad0bdb32f39a6c4a4/src/modules/EventList/index.tsx#L39